development.ini

```diff
@@ -1,11 +1,21 @@
 [server:main]
+# FOR DEVELOPMENT USE
 use = egg:Paste#http
 host = 0.0.0.0
 port = 8088
 
+# FOR PRODUCTION USE
```

Comment:
Config file options that are for develop or production use should be set correctly in their own files named by domain (production.conf, development.conf)